src/main/docs/guide/kafkaStreams.adoc

```diff
@@ -109,20 +96,13 @@ The above configuration sets the `num.stream.threads` setting of the Kafka `Stre
 
 You can then inject an `api:configuration.kafka.streams.ConfiguredStreamBuilder[]` specifically for the above configuration using `jakarta.inject.Named`:
 
-[source,java]
-----
-@Named("my-other-stream")
-KStream<String, String> myOtherKStream(@Named("my-other-stream") ConfiguredStreamBuilder builder)  {...}
-----
+snippet::io.micronaut.kafka.docs.streams.WordCountStream[tags="clazz, myOtherStream"]
```

Comment:
```suggestion
snippet::io.micronaut.kafka.docs.streams.WordCountStream[tags="myOtherStream"]
```

Unless you have a specific reason in mind to make this snippet longer than it was, I would leave out `clazz` to keep it similar to the original example:

```java
@Named("my-other-stream")
KStream<String, String> myOtherKStream(@Named("my-other-stream") ConfiguredStreamBuilder builder)  {...}
```